src/main/java/com/celements/model/classes/fields/BooleanField.java

```diff
@@ -20,8 +18,8 @@
     private String displayType;
     private Integer defaultValue;
 
-    public Builder(@NotNull DocumentReference classRef, @NotNull String name) {
-      super(classRef, name);
+    public Builder(@NotNull String className, @NotNull String name) {
```

Comment:
please do not use String representations for document full names. We are working about 10 years to get rid of all String typed full names. Use DocumentReferences or EntityReferences instead.